extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts

```diff
@@ -139,26 +139,52 @@ export class ArcGisFeatureProvider extends ArcGISImageryProvider {
       }
     }
 
+    // Parse server version
+    let majorVersion: number|undefined;
+    if (this.serviceJson?.currentVersion) {
+      try {
+        majorVersion = Math.trunc(this.serviceJson?.currentVersion);
```

Comment:
does this actually throw? guess it doesnt hurt to wrap incase of malformed serviceJson but itd have to be pretty screwed up

Reply:
yea I dont know, just wanted to avoid any mess.